RDVFSharp/Commands/General/Register.cs

```diff
@@ -20,24 +20,21 @@ public async Task Execute(string character, IEnumerable<string> args, string cha
             var fighter = await Plugin.DataContext.Fighters.FindAsync(character);
             if (fighter != null)
             {
-                throw new FighterAlreadyExists(character);
+                Plugin.FChatClient.SendMessageInChannel("You are already registered!", channel);
             }
 
             int[] statsArray;
 
-            try
+            
```

Comment:
You should definitely not remove the try/catch here! It can fail in many ways. Keep the "throw new Exception();" but in the catch, send the message instead of throwing another exception